lib/src/main/java/graphql/nadel/NadelSchemas.kt

```diff
@@ -136,14 +136,15 @@ data class NadelSchemas constructor(
 
         fun build(): NadelSchemas {
             require(overallSchemaReaders.isNotEmpty()) { "Nadel schemas must not be empty" }
-            require(underlyingSchemaReaders.isNotEmpty()) { "Underlying schemas must not be empty" }
+            require(underlyingSchemaReaders.isNotEmpty() || underlyingTypeDefs.isNotEmpty()) { "Underlying schemas must not be empty" }
```

Comment:
Fixes check if you only supply type defs and no readers. There's a new test for this below too.